docs/en/writing-running-appium/tutorial/swipe/simple-screen.md

```diff
@@ -1,9 +1,11 @@
 ## Simple Screen swipe
 
-Phone applications have interface. Interface, in most cases, blocks your
-swipe actions when you start from it. Historically it uses header and/or
-footer. To make screen swipe more reliable - it is better to start swipe
-action from center of screen.
+Start of swipe is important. We can met following elements:
+- application interface/menu e.g. header or footer menu
+- elements that waiting tap and do not pass touch to scroll view
```

Comment:
Maybe I could help to rephrase this requirement?

Reply:
it is hard to understand while reading 5 comments on 2 sentences. why not just suggest whole 2 sentences?

Reply:
Sorry, I'm trying to compile significant parts, but it looks like it just does not work for this particular paragraph :(

Write it in your mother tongue and I'll try to translate.